### ISSUE TITLE
Licences for artwork

### ISSUE THREAD

**Issue by robinpaulson** [martykan/forecastie#450]
At present, our icons, etc. are by default covered by GPL3+, the same as the rest of the project. GPL variants are not great for art though, Larry Lessig has written lots on this, he developed CC as a response. So, what should we choose?

I think CC-BY-SA 4.0 is the closest in spirit to GPL3+. Suggestions requested.

Note: no licence war arguments in this issue please (BSD vs. GPL, etc.), Tomáš has already determined the type of licence (strong copyleft, etc.), we are merely interpreting and implementing that decision in a way appropriate to artistic artefacts.

**Comment by sotpapathe** [comment]
The consent of the copyright holder would be needed to change the license of the icons. Some were commited by Tomáš  (`forecastie/icon.png`) and others by Andrew Mattie (`forecastie/app/src/main/ic_launcher-web.png`, `forecastie/app/src/main/assets/images/marker-icon-2x.png`). It is also not clear whether they created these icons themselves or got them from somewhere under a license compatible with the GPLv3. Same goes for the fonts.

**Comment by robinpaulson** [comment]
@sotpapathe Yes, of course that is correct. I'll contact Tomáš and ask. Good spot on Andrew Mattie's work, I (hastily) assumed Tomáš had contributed them all.

**Comment by sotpapathe** [comment]
The list of people who contributed .png files:
- Igor Calì
  - 7fe2d39678c6c0880912f7a77692348077da8042
    - app/src/main/res/drawable-nodpi/widget_preview_time_classic.png

- Andrew Mattie
  - 853042144481990e31ed7c85effd71e0ae0fe981
    - app/src/main/ic_launcher-web.png
    - app/src/main/res/mipmap-hdpi/ic_launcher_round.png
    - app/src/main/res/mipmap-mdpi/ic_launcher_round.png
    - app/src/main/res/mipmap-xhdpi/ic_launcher_round.png
    - app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png
    - app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png
  - 9a896edefa3b7acf6e62d59158b65430780df257
    - app/src/main/assets/images/marker-icon-2x.png
    - app/src/main/assets/images/marker-shadow.png

- François Magimel
  - c0a13dda4a14bdfd05a4b542a40ede2da96ccadd
    - app/src/main/res/drawable-nodpi/widget_preview_simple.png

- Alexander Bikadorov
  - 8a5b43cdce6deaf02dfe433e8856bb460a3a6a09
    - app/src/main/res/drawable-nodpi/widget_preview_time.png

- Tomas Martykan
  - 28620360bdfe517b0fda5eb5c01684f0be700f18
    - app/src/main/res/drawable-*
  - 832a35eeec12e21d552a5acff7167e62d50636a8
    - app/src/main/res/drawable-*
  - 1046cdbefa4e3a445450d380900c0a2623d23750
    - icon.png
  - 1b5493c8bf500262c411042f233a9bcdcbd989a7
    - app/src/main/res/drawable-*
  - fd327e9a0c51403e76fcd9200481f77330d544ef
    - app/src/main/res/drawable-*
    - app/src/main/res/mipmap-*

- SpiritCroc
  - 0a7652c73051aae4e05f2828e6d0cb7ece766aef
    - app/src/main/res/drawable-nodpi/widget_preview.png
    - app/src/main/res/drawable-nodpi/widget_preview_time.png

Generated using `git log --summary $(find . -iname '*.png' | grep -i -F .png)`

**Comment by robinpaulson** [comment]
Good git mastery there, I guess there's some contacting work to be done!
